application/lib/console.dart

```diff
@@ -60,7 +59,7 @@ class _ConsoleState extends State<Console> {
   }
 
   void _send() {
-    widget.game.sendMessage(_input.text).catchError((Object error) { }, test: (Object error) => error is ConnectionLostException);
+    widget.game.sendMessage(_input.text).catchError((Object error) => 'ignorethis', test: (Object error) => error is ConnectionLostException);
```

Comment:
i meant literally `''`, i don't want `ignorethis` to get compiled into the binary :-)

Reply:
fixed